src/Stages.js

```diff
@@ -1,11 +1,15 @@
 import React, { Component } from 'react';
 import './styles/App.css';
+import StagesCards from './StagesCards.js';
 
 class Stages extends Component {
   constructor() {
     super();
     this.state = {
-      data: []
+      stages: [],
+      card: '',
+      currentStage: '',
+      nextStage: ''
```

Comment:
Not sure we will need the nextstage stored in state if we are just going to render the image in a center screen and not have left and right arrows.